src/sync/installation.ts

```diff
@@ -356,22 +387,14 @@ const handleBackfillError = async (err,
 
 	// Continue sync when a 404/NOT_FOUND is returned
 	if (isNotFoundError(err, logger)) {
-		const edgesLeft = []; // No edges left to process since the repository doesn't exist
-		await updateJobStatus(data, edgesLeft, task, repositoryId, logger, scheduleNextTask);
+		await ignoreCurrentRepo();
 		return;
 	}
 
-	// TODO: add the jiraHost to the logger with logger.child()
-	const host = subscription.jiraHost || "none";
```

Comment:
There is no need to extract host from subscription and adding it to the log anymore. Jira Hostname is already added to the logger in the backfill message handler.